### DISCUSSION TITLE
Open Monitor Report

### DISCUSSION THREAD

**Discussion by gherbstman** [LibreHardwareMonitor/LibreHardwareMonitor#1051] [Q&A]
Is there an Open Monitor Report component available to dump a complete report? If not, how would I go about using the lib from PowerShell?

**Comment by visormail** [comment]
That's my question too, the report component with the OpenHardwareMonitorReport.exe ReportToConsole command, in the original version that was way more detailed than the gui. But I see no asnwer for this.
